Configuration/PyReleaseValidation/python/relval_2026.py

```diff
@@ -34,7 +34,7 @@
 numWFIB.extend([39034.0]) #2026D87
 numWFIB.extend([39434.0,39434.911,39434.103]) #2026D88 DDD XML, DD4hep XML, aging
 numWFIB.extend([39661.97]) #2026D88 premixing stage1 (NuGun+PU)
-numWFIB.extend([39434.5,39434.9,39434.501,39434.502]) #2026D88 pixelTrackingOnly, vector hits, Patatrack local reconstruction on CPU, Patatrack local reconstruction on GPU
+numWFIB.extend([39434.5,39434.9,39434.502]) #2026D88 pixelTrackingOnly, vector hits, Patatrack local reconstruction on CPU, Patatrack local reconstruction on GPU
```

Comment:
should the comment here be updated as well?

Reply:
I would actually prefer to keep the two configuration (`.501` and `.502`) for the time being

Reply:
No problem for me to postpone it. Still would drop it at a certain point (: